export/wiki/spawn_groups/remaps.py

```diff
@@ -0,0 +1,39 @@
+from typing import List, Optional
+
+from automate.hierarchy_exporter import ExportModel, Field
+from ue.asset import UAsset
+from ue.utils import sanitise_output
+
+__all__ = ['convert_npc_remaps']
+
+
+class ClassRemap(ExportModel):
+    from_bp: str = Field(alias="from")
+    to: Optional[str] = None
```

Comment:
This looks reusable. Please move somewhere more central. Maybe create `export.models` or similar?